packages/quantic/playwright/page-object/searchObjectWithDidYouMean.ts

```diff
@@ -0,0 +1,91 @@
+import {Page} from '@playwright/test';
+import {SearchObject} from './searchObject';
+
+export type WordCorrectionData = {
+  correctedWord: string;
+  originalWord: string;
+  length: number;
+  offset: number;
+};
+
+export type DidYouMeanData = {
+  correctedQuery: string;
+  wordCorrections: WordCorrectionData[];
+};
+
+export type DidYouMeanNextData = {
+  correctedQuery: string;
+  wordCorrections: WordCorrectionData[];
+  originalQuery: string;
+};
```

Comment:
```suggestion
export type DidYouMeanNextData = DidYouMeanData & {
  originalQuery: string;
};
```

what about this?

Reply:
Those are two different API response structure, not sure I want to link them together.

It's not visible here, but the `DidYouMeanData` goes into the `searchResponse.queryCorrections` and the `DidYouMeanNextData` goes into the `searchResponse.queryCorrection` notice the missing `s`.